app/src/main/java/com/android/voyageur/ui/profile/ProfileScreen.kt

```diff
@@ -1,31 +1,155 @@
+                    }
+                }
+            }
+        } else {
+            // Navigate to AUTH if no current user is logged in
+            navigationActions.navigateTo(Route.AUTH)
+        }
+    }
+
+    // Main Scaffold layout for ProfileScreen with Bottom Navigation
+    Scaffold(
+        modifier = Modifier.testTag("profileScreen"),
+        bottomBar = {
+            BottomNavigationMenu(
+                onTabSelect = { route -> navigationActions.navigateTo(route) },
+                tabList = LIST_TOP_LEVEL_DESTINATION,
+                selectedItem = navigationActions.currentRoute(),
+            )
+        },
+        content = { paddingValues ->
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(paddingValues)
+                    .testTag("profileScreenContent"),
+                contentAlignment = Alignment.Center
+            ) {
+                if (isSigningOut) {
+                    CircularProgressIndicator(modifier = Modifier.testTag("signingOutIndicator"))
+                } else if (isLoading) {
+                    CircularProgressIndicator(modifier = Modifier.testTag("loadingIndicator"))
+                } else {
+                    user?.let { userData ->
+                        ProfileContent(
+                            userData = userData,
+                            onSignOut = { isSigningOut = true }
+                        )
+                    } ?: run {
+                        Text("No user data available", modifier = Modifier.testTag("noUserData"))
+                    }
+                }
+            }
+        }
+    )
 }
+
+@Composable
+fun ProfileContent(userData: User, onSignOut: () -> Unit) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+            .testTag("profileContent"),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        // Display the profile picture if available
+        if (userData.profilePicture.isNotEmpty()) {
+            Image(
+                painter = rememberAsyncImagePainter(model = userData.profilePicture),
+                contentDescription = "Profile Picture",
+                modifier = Modifier
+                    .size(128.dp)
+                    .clip(CircleShape)
+                    .testTag("profilePicture")
+            )
+        } else {
+            Icon(
+                imageVector = Icons.Default.AccountCircle,
+                contentDescription = "Default Profile Picture",
+                modifier = Modifier
+                    .size(128.dp)
+                    .testTag("defaultProfilePicture")
+            )
+        }
+
+        Spacer(modifier = Modifier.height(16.dp))
+
+        // Display user name and email
+        Text(
+            text = "Name: ${userData.name.takeIf { it.isNotEmpty() } ?: "No name available"}",
```

Comment:
If the name is empty, just display an empty cell

Reply:
Also, the I updated the CI file / ruleset for branches so if the tests won't start on your commits, but will work fine on new branches. After you address your changes, please add run the tests again and add an SS if they pass.

Reply:
I see your suggestion about displaying an empty cell if the name is missing, but I think showing a placeholder like "No name available" might offer a better user experience. It provides clearer feedback to the user that their profile is incomplete, which could encourage them to update their information. Displaying an empty cell could be confusing or make the UI feel broken, especially if users expect all fields to be filled in. By showing a placeholder, we maintain consistency and clarity across the profile interface.

That said, if you still prefer the empty cell approach, I can make that change! 
Also, thanks for the heads-up about the CI ruleset update! I'll make sure to push my changes on a new branch so that the tests can run correctly under the updated configuration.

Reply:
I’ve updated my tests from the main branch, and everything passes now. Thanks for the heads-up on the CI changes! I ran the tests after addressing my updates, and they all passed successfully.